ci-operator/step-registry/ipi/conf/vsphere/lb/external/ipi-conf-vsphere-lb-external-commands.sh

```diff
@@ -10,6 +9,9 @@ if [[ -z "${LEASED_RESOURCE}" ]]; then
   exit 1
 fi
 
+LEASE_NUMBER=$((${LEASED_RESOURCE//[!0-9]/}))
+declare LEASE_NUMBER
```

Comment:
I guess we do not need the `declare` line, if we already defined it on the prior line.

Reply:
ok, a mistake. will delete.